src/app/zwiazek/historia-zwiazku/data.ts

```diff
@@ -0,0 +1,51 @@
+interface Paragraph {
+  paragraph: string | string[];
+}
+export interface Subsection extends Paragraph {
+  subtitle?: string;
+  bulletpoints?: string[];
+}
+
+interface ArticleStructure {
+  mainArticle: Paragraph;
+  subsections: {
+    establishment: Subsection;
+    goals: Subsection;
+    development: Subsection;
+  };
+  addendum: Paragraph;
+}
+export const unionData: ArticleStructure = {
```

Comment:
You could export 3 separate objects if needed, even more.

You are destructuring it in the page after all 😉 
> `const { mainArticle, subsections, addendum } = unionData;`